RecoLocalMuon/GEMRecHit/python/gemRecHits_cfi.py

```diff
@@ -9,3 +9,6 @@
     #maskFile = cms.FileInPath("RecoLocalMuon/GEMRecHit/data/maskedStrips.txt"),
     #deadFile = cms.FileInPath("RecoLocalMuon/GEMRecHit/data/deadStrips.txt")
     )
+
+from Configuration.Eras.Modifier_phase2_GEM_cff import phase2_GEM
+phase2_GEM.toModify(gemRecHits, gemDigiLabel = cms.InputTag("simMuonGEMDigis"))
```

Comment:
Should we expect to see the difference in the test when you use simMuonGEMDigis instead of muonGEMDigis? What is done currently w/o this PR?

Reply:
No, there shouldn't be a difference. The muonGEMDigis are created by sending the simMuonGEMDigis through the packer/unpacker chain, and this should essentially act as a test of the packer. Since for phase2, the RAW format isn't established yet and we are currently working on getting the unpacker setup with the MWGR data now coming from the GEMs. We want to skip the these steps for phase2 and just use the simMuonGEMDigis.